http-server/configuration-production.yml

```diff
@@ -3,6 +3,8 @@ githubApiToken: ${GITHUB_API_TOKEN}
 # if we are using any stubbing, we'll assert that this value is false.
 # this is to help guarantee that we do not accidently use test configuration in prod.
 prod: true
+logRequestAndResponses: false
+logSqlStatements: false
```

Comment:
I'm a bit on the fence if SQL statements should be logged in prod. The parameters are not logged, so it would be 'safe', but I think in the end it would become very noisy.

Reply:
I believe logging makes sense when using something like Hibernate where we don't control the SQL that gets sent to the database.
But that's currently not the case, right?